dotCMS/src/test/java/com/dotcms/util/pagination/ContentTypesPaginatorTest.java

```diff
@@ -0,0 +1,35 @@
+package com.dotcms.util.pagination;
+
+import com.dotmarketing.portlets.structure.business.StructureAPI;
+import com.dotmarketing.util.PaginatedArrayList;
+import com.google.common.collect.ImmutableMap;
+import com.liferay.portal.model.User;
+import org.junit.Test;
+import java.util.Map;
+
+import static org.mockito.Mockito.mock;
+
+/**
+ * Test of {@link ContentTypesPaginator}
+ */
+public class ContentTypesPaginatorTest {
```

Comment:
I recently created an IT called ContentTypesPaginatorTest in the same package....please check if it doesn't generate a conflict when Jenkins runs tests.